.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           echo "$WEBAPP_GIT_COMMIT"
 
           FILE_DIST=dist.tar.gz
-          until curl --max-time 5 -f -o $FILE_DIST https://pr-builds.mattermost.com/mattermost-webapp/commit/${WEBAPP_GIT_COMMIT}/mattermost-webapp.tar.gz; do echo waiting for webapp build; sleep 110; done;
+          until curl --max-time 75 -f -o $FILE_DIST https://pr-builds.mattermost.com/mattermost-webapp/commit/${WEBAPP_GIT_COMMIT}/mattermost-webapp.tar.gz; do echo waiting for webapp build; sleep 5; done;
```

Comment:
Just to confirm my understanding: if the the build is missing curl is going to exit right away so what's the purpose of the `--max-time` value?

Reply:
https://app.circleci.com/pipelines/github/mattermost/mattermost-server/27571/workflows/276bb842-43fe-4bef-b768-f3d62634004b/jobs/193598 It actually polls the file, the max-times we tell it to.

Reply:
But the poll frequency is defined by the sleep value, right? `--max-time` is just a timeout for the curl command which in the case of a 404/403 should be almost instant to return so I don't understand how it's needed.

Reply:
You are completely right. I readjusted the options of curl. Please take a look.